Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -286,38 +286,44 @@ theorem completeSpace_iff_isComplete_range {f : α → β} (hf : UniformInducing
     CompleteSpace α ↔ IsComplete (range f) := by
   rw [completeSpace_iff_isComplete_univ, ← isComplete_image_iff hf, image_univ]
 
+alias ⟨_, UniformInducing.completeSpace⟩ := completeSpace_iff_isComplete_range
+
 theorem UniformInducing.isComplete_range [CompleteSpace α] {f : α → β} (hf : UniformInducing f) :
     IsComplete (range f) :=
   (completeSpace_iff_isComplete_range hf).1 ‹_›
 
+theorem UniformInducing.completeSpace_congr {f : α → β} (hf : UniformInducing f)
```

Comment:
Can you add a docstring comparing this to `_root_.completeSpace_congr`, and perhaps one in the other direction too?